src/gui/folder.cpp

```diff
@@ -897,7 +897,7 @@ void Folder::correctPlaceholderFiles()
     }
 }
 
-void Folder::setSyncOptions()
+SyncOptions Folder::initializeSyncOptions() const
```

Comment:
A return type's been added but the method doesn't actually return any `SyncOptions`, is this intentional?